### PR TITLE
NEWS.md: add notes on 0.6.1

### PR DESCRIPTION
**Pull request by johanmalm**
Prior to release:

- [x] #725

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
...
+- Add `<regions>` config option allowing the definition of regions to which
+  windows can be snapped by keeping a keyboard modifier pressed while dragging
+  or by using the SnapToRegion action. Written-by: @Consolatis
+- Add `<SnapToRegion>` action. Written-by: @Consolatis
+- Add `<Kill>` action to send SIGTERM to a client process. Written-by: @bi4k8
+- Add config option `<core><reuseOutputMode>` to support flicker free boot
+  (issue #724). Written-by: @Consolatis
+- Enable single-pixel-buffer-v1
+- Improve debugging by adding argument to Debug action adding a label for the
+  layer popup tree. Written-by: @Consolatis
```
Comment (Consolatis) [inline NEWS.md:40]: This seems to miss an `and`. Could also be removed completely, I don't think its that relevant for labwc users.

File: NEWS.md [hunk 2]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
...
+- Support theme setting override by reading `<config-dir>/themerc-override`
+  where `<config-dir>` is normally $HOME/.config/labwc can be other locations
```
Comment (Consolatis) [inline NEWS.md:42]: Seems to miss an `but`.

Could also be reduced to just `Support single theme setting overrides by reading `<config-dir>/themerc-override`.

**Comment by Consolatis** [comment]
Also: `0.7.0` or `0.6.1`?

Currently we are somewhat connected to the wlroots releases, e.g. `0.5.x` -> `wlroots 0.15.x` and `0.6.0` > `wlroots 0.16.x`.
I think I'd prefer `0.6.1` for that reason, although one could also argue that new features have been implemented which deserve a new minor version.

File: NEWS.md [hunk 3]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
...
+- On un-fullscreen, restore SSD before applying previous geometry to avoid
+  rendering offscreen in some instances. Written-by: @Consolatis
+- Allow snapping to the same edge. Thanks-to: @Consolatis and @Flrian
+- Send enter event when new layer surface appears under pointer. Issue #667
+- Prevent re-focus for always-on-top views when switching workspaces.
+  Written-by: @Consolatis
+- Make sure a default libinput category always exists to avoid devices not
+  being configured is some insances. Written-by: @jlindgren90
+- Update cursor if it is within the OSD area when OSD appears/disappears.
+  Written-by: @Flrian
```
[the remark this answers is not in the document]
Comment (johanmalm) [reply NEWS.md:71]: Fixed. Thanks

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Removed.

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: Reduced.

**Comment by johanmalm** [comment]
> Also: `0.7.0` or `0.6.1`?
> Currently we are somewhat connected to the wlroots releases, e.g. `0.5.x` -> `wlroots 0.15.x` and `0.6.0` > `wlroots 0.16.x`. I think I'd prefer `0.6.1` for that reason, although one could also argue that new features have been implemented which deserve a new minor version.

Have gone for `0.6.1`.

File: NEWS.md [hunk 4]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
...
+  windows can be snapped by keeping a keyboard modifier pressed while dragging
+  or by using the SnapToRegion action. Written-by: @Consolatis
+- Add `<SnapToRegion>` action. Written-by: @Consolatis
+- Add `<Kill>` action to send SIGTERM to a client process. Written-by: @bi4k8
+- Add config option `<core><reuseOutputMode>` to support flicker free boot
+  (issue #724). Written-by: @Consolatis
+- Enable single-pixel-buffer-v1
+- Support theme setting override by reading `<config-dir>/themerc-override`
+- Scale down SSD button icons if necessary to allow using larger for high and
+  mixed DPI usecases. Issue #609. Written-by: @Consolatis
```
Comment (Consolatis) [inline NEWS.md:41]: [..] to allow using larger +ones+ for high [..]

File: NEWS.md [hunk 5]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
...
+- Handle client request for layer-change
+- Support setting color of client menu buttons. @Flrian
```
Comment (Consolatis) [inline NEWS.md:43]: Nit: `Written-by:`

File: NEWS.md [hunk 6]
```diff
@@ -19,6 +20,70 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## 0.6.1 - 2023-01-27
```
Comment (Consolatis) [inline NEWS.md:23]: Nit: The date doesn't match with the list above

Comment (johanmalm) [reply NEWS.md:23]: Fixed.

File: NEWS.md (hunk 5 above)
Comment (johanmalm) [reply NEWS.md:42]: Fixed.

**Pull request merged**